app/resources/en/translation/tasklist.json

```diff
@@ -36,7 +36,7 @@
   "taskNotStarted": "Start",
   "taskStarted": "Continue",
   "taskComplete": "Check your answers",
-  "taskUnavailable": "You must complete the previous step before you can start this",
-  "taskCompleteTag": "Completed",
+  "taskUnavailable": "Can&rsquo;t start yet",
+  "checkYourAnswers": "Completed",
```

Comment:
It would be better to switch the keys for `taskComplete` and `checkYourAnswers` around as the content for `taskComplete` is content for the check your answers link and `checkYourAnswers` is content for the task complete tag.